ci/common.py

```diff
@@ -0,0 +1,35 @@
+# coding=utf-8
+import os
+import subprocess
+
+
+def log(msg, expand=False):
+    if(expand):
```

Comment:
According to the style guide at python.org/dev/peps/pep-0008/ (and what we use in the GDK codebase) I would prefer if could use spaces after `if` definitions, for all cases in all files please. 
```suggestion
    if (expand):
```